src/components/HURUmap/Chart/utils.js

```diff
@@ -1,4 +1,4 @@
-const idify = (string) => {
+export const idify = (string) => {
```

Comment:
```suggestion
export function idify(string) {
```

Why oh why do people love arrow functions so much?